lib/autoload/polyglot/language/python.rb

```diff
@@ -0,0 +1,25 @@
+class Polyglot::Language::Python < Polyglot::Language
+  class Python2Point7 < Polyglot::Language::Python
+    class << self
+      def instance
+        root
```

Comment:
Except when user-defined languages in future where say CS1101S uses JediScript which is just JavaScript, then it would be an instance of JavaScript, with the parent as JavaScript.Instance.